package.json

```diff
@@ -18,6 +20,7 @@
     "demo:ie": "yarn demo -c ie",
     "ssr": "yarn ssr-app:lint && yarn ssr-app:build && yarn ssr-app:e2e",
     "changelog": "conventional-changelog --preset angular --infile CHANGELOG.md --same-file --release-count 1",
+    "changelog:patch": "git add CHANGELOG.md && git commit --amend --no-edit && git tag -f $npm_package_version",
```

Comment:
Is `$npm_package_version` automatically there thanks to nom cli ?

Reply:
yeah: https://docs.npmjs.com/misc/scripts#packagejson-vars

The only issue is that on Windows one should use `%$npm_package_version%

Reply:
Even on WSL or Cygwin ? `% ... %` sounds more like CMD or PowerShell

Reply:
Yes, I think you're right